src/services/form-validation.ts

```diff
@@ -668,6 +668,15 @@ export function checkFileNameIsValid(
   )
 }
 
+export function checkFileNameHasExtension(
```

Comment:
IMO we call this 
```checkFileNameExtensionIsValid```
since we are not explicitly checking if it has an extension, we are checking if the file name/extension is ok based on whether it has an extension and whether we allow extensions